src/test/utils/expression/randomAst.ts

```diff
@@ -0,0 +1,260 @@
+import fc from "fast-check";
+import {
+    AstBoolean,
+    AstConditional,
+    AstExpression,
+    AstFieldAccess,
+    AstId,
+    AstInitOf,
+    AstMethodCall,
+    AstNull,
+    AstNumber,
+    AstOpBinary,
+    AstOpUnary,
+    AstStaticCall,
+    AstString,
+    AstStructFieldInitializer,
+    AstStructInstance,
+} from "../../../ast/ast";
+import { dummySrcInfo } from "../../../grammar/src-info";
+
+function dummyAstNode<T>(
+    generator: fc.Arbitrary<T>,
+): fc.Arbitrary<T & { id: number; loc: typeof dummySrcInfo }> {
+    return generator.map((i) => ({
+        ...i,
+        id: 0,
+        loc: dummySrcInfo,
+    }));
+}
+
+function randomAstBoolean(): fc.Arbitrary<AstBoolean> {
+    return dummyAstNode(
+        fc.record({
+            kind: fc.constant("boolean"),
+            value: fc.boolean(),
+        }),
+    );
+}
+
+function randomAstString(): fc.Arbitrary<AstString> {
+    const escapeString = (s: string): string =>
+        s.replace(/\\/g, "\\\\").replace(/"/g, '\\"');
+
+    return dummyAstNode(
+        fc.record({
+            kind: fc.constant("string"),
+            value: fc.string().map((s) => escapeString(s)),
+        }),
+    );
+}
+
+function randomAstNumber(): fc.Arbitrary<AstNumber> {
+    const values = [
+        ...Array.from({ length: 10 }, (_, i) => [0n, BigInt(i)]).flat(),
+        ...Array.from({ length: 256 }, (_, i) => 1n ** BigInt(i)),
+    ];
+
+    return dummyAstNode(
+        fc.record({
+            kind: fc.constant("number"),
+            base: fc.constantFrom(2, 8, 10, 16),
```

Comment:
Reuse the corresponding array defined in AST when https://github.com/tact-lang/tact/issues/1465 is solved